### PR TITLE
RSDK-862: add a default max_rpm to fake motor

### PR DESCRIPTION
**Pull request by stevebriskin**

### CODE REVIEW

File: components/motor/fake/motor.go [hunk 1]
```diff
@@ -21,7 +21,10 @@ import (
 	rdkutils "go.viam.com/rdk/utils"
 )
 
-const modelName = "fake"
+const (
+	modelName     = "fake"
+	defaultMaxRpm = 1000
```
Comment (penguinland) [inline components/motor/fake/motor.go:26]: The max_rpm in fake.json used to be 100, and the new value is 10 times larger. Is that change on purpose? I doubt it affects anything important, but it does seem a little unusual.

**Review by penguinland — APPROVED** [review]
LGTM!

**Review by randhid — APPROVED** [review]

File: components/motor/fake/motor.go (hunk 1 above)
Comment (stevebriskin) [reply components/motor/fake/motor.go]: updated to a 100
